packages/terra-form/src/components/Fieldset.jsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/jsx-boolean-value, jsx-a11y/label-has-for */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Field.scss';
+
+const propTypes = {
+  /**
+   * Children the Field contains
+   */
+  children: PropTypes.node,
+  /**
+   * Error message for when the input is invalid
+   */
+  error: PropTypes.node,
+  /**
+   * Help element to display with the input
+   */
+  help: PropTypes.node,
+  /**
+   * The value for the htmlFor property on the label
+   */
+  htmlFor: PropTypes.string,
+  /**
+   * Determines whether the field in an inline field
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Legend for the input group
+   */
+  legend: PropTypes.string,
+};
+
+const defaultProps = {
+  children: null,
+  error: null,
+  help: null,
+  htmlFor: null,
+  isInline: false,
+  legend: null,
+};
+
+const Fieldset = ({ children, error, help, htmlFor, isInline, legend, ...customProps }) => {
+  const fieldsetClasses = classNames(
+    'terra-Form-field',
```

Comment:
Would we want a specific css class for fieldset to indicate the difference between a field component and a fieldset component?